media-sound/qmmp/qmmp-9999.ebuild

```diff
@@ -5,11 +5,10 @@ EAPI=8
 
 inherit cmake xdg
 
-DESCRIPTION="Qt5-based audio player with winamp/xmms skins support"
+DESCRIPTION="Qt-based audio player with winamp/xmms skins support"
 HOMEPAGE="https://qmmp.ylsoftware.com"
 if [[ ${PV} != *9999* ]]; then
-	SRC_URI="https://qmmp.ylsoftware.com/files/${P}.tar.bz2
-		https://downloads.sourceforge.net/${PN}-dev/files/${P}.tar.bz2"
+	SRC_URI="https://qmmp.ylsoftware.com/files/${PN}/$(ver_cut 1-2)/${P}.tar.bz2"
```

Comment:
I did not quite understand dropping the sourceforce mirror, so will keep that on merge.